spec/features/brexit_checker/question_results_spec.rb

```diff
@@ -32,25 +32,30 @@
 
   def then_i_see_citizen_and_business_results
     then_i_should_see_the_results_page
+    and_i_should_see_citizen_actions_are_grouped
     and_i_should_see_the_citizens_action_header
     and_i_should_see_the_business_action_header
     and_i_should_see_a_pet_action
     and_i_should_see_a_tourism_action
+    and_the_tourism_link_should_have_tracking_analyitics
   end
 
   def then_i_see_business_results_only
     then_i_should_see_the_results_page
     expect(page).to_not have_content I18n.t!("brexit_checker.results.audiences.citizen.heading")
     and_i_should_see_the_business_action_header
     and_i_should_see_a_ce_mark_action
+    and_the_ce_mark_link_should_have_tracking_analyitics
   end
 
   def then_i_see_citizens_results_only
     then_i_should_see_the_results_page
     and_i_should_see_the_citizens_action_header
     expect(page).to_not have_content I18n.t!("brexit_checker.results.audiences.business.heading")
```

Comment:
Same comment here about being consistent in not using expectations within scenarios.

Reply:
done!